.clangd

```diff
@@ -0,0 +1,2 @@
+CompileFlags:
+  Add: [-DLSP, -Wno-missing-declarations, -Wno-typedef-redefinition, -Wno-ignored-attributes, -xc]
```

Comment:
maybe add the clang format as well ([kernel docs](https://www.kernel.org/doc/html/latest//dev-tools/clang-format.html))?

Reply:
added kernel config